### ISSUE TITLE
Error installing Bugsnag while installing with cocoapods

### ISSUE THREAD

**Issue by ebsaral** [bugsnag/bugsnag-cocoa#49]
```
Installing Bugsnag (4.0.6)

[!] Error installing Bugsnag
[!] /usr/bin/git submodule update --init

Submodule 'KSCrashModule' (git@github.com:bugsnag/KSCrash.git) registered for path 'KSCrashModule'
Cloning into 'KSCrashModule'...

Warning: Permanently added the RSA host key for IP address '192.30.252.128' to the list of known hosts.

fatal: reference is not a tree: 76836a36603fee7475731c8826b62dc1a6c47441

Unable to checkout '76836a36603fee7475731c8826b62dc1a6c47441' in submodule path 'KSCrashModule'
```

When I run pod install, I ran into this error and this happens a lot. When I change Bugsnag to version 4.0.3, it installs without any error.

Anyone knows what am I doing wrong or what's wrong?

**Comment by isair** [comment]
+1

This error doesn't happen on my machine, alas it does in TravisCI and CircleCI.

**Comment by xinsight** [comment]
I ran into the same problem, but I think it was fixed. Use Bugsnag 4.0.7?

https://github.com/bugsnag/bugsnag-cocoa/issues/46

**Comment by isair** [comment]
Updated, and I can confirm that the problem is now gone on the CI servers.

**Comment by jessicard** [comment]
This seems to be fixed in 4.0.7. Let me know if there's any more issues with this.
